rpc/block_svc_routines.c

```diff
@@ -41,7 +41,7 @@
                                 "'%s' ls | grep -e tpg -e '%s' | grep -B1 '%s' | grep -o 'tpg\\w'"
 # define   GB_CHECK_PORTAL      "targetcli /iscsi/" GB_TGCLI_IQN_PREFIX \
                                 "'%s' ls | grep '%s' > " DEVNULLPATH
-# define   GB_SAVECONFIG_CHECK  "grep -m 1 '\"name\": \"%s\",' " GB_SAVECONFIG " > " DEVNULLPATH
+# define   GB_SAVECONFIG_CHECK  "grep -m 1 \"%s\",' " GB_SAVECONFIG " > " DEVNULLPATH
```

Comment:
@lxbsz 
we were searching for ```"name": "myblockname",``` syntax before. Can you tell me what exactly this change is looking for from the saveconfig.json  ?

Reply:
@pkalever 

Did you see the commit comment :

```
    delete: use gbid instead of block_name to do the saveconfig check
    
    This will be one problem if we create 2 block targets by using the
    same NAME in 2 different volumes. The second time it will fail and
    won't be loaded and no any info in the saveconfig.json, but it will
    still could find it which is the StorageObject created by the first
    creation, the the rollback will fail like:

```

That means roll back will delete the SO created by the first creation, so here will use the GBID instead of the NAME to do the search.

Thanks

Reply:
@lxbsz I saw it already thanks!

I mean to ask what exactly you are grepping in the saveconfig file ?

Reply:
The SO.

Here we may need to grep the :

"config": "glfs/dht@localhost/block-store/bf4d1098-fa85-4057-9932-f2dec0dfe647",

or 

"wwn": "bf4d1098-fa85-4057-9932-f2dec0dfe647"

？

Reply:
it won't match wwn as it doesn't end with coma.

It should match configstr. I thin that's enough for us.

Thanks for the details.